rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/AppContextListener.java

```diff
@@ -0,0 +1,23 @@
+package io.narayana.lra.coordinator.api;
+
+import com.arjuna.ats.arjuna.recovery.RecoveryManager;
+import io.narayana.lra.coordinator.internal.Implementations;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+
+// start the recovery manager early so that we can start recovering in doubt LRAs
```

Comment:
I don't know what you mean. But the code works as follows:

- when the war/jax-rs application get's loaded we start the recovery module
- otherwise we start the recovery module on demand (ie whenever a request is made on any of the resources that implement narayana LRA)

Reply:
Oh I see what you mean, I forgot to push my web.xml

Reply:
I was thinking in way that before there was the LRARecoveryModule configured during the Recovery manager startup. But I found now the jboss-properties.xml defines the module to be configured. I do understand now. Thanks for elaboration.